src/main/java/github/scarsz/discordsrv/api/events/AchievementMessagePostProcessEvent.java

```diff
@@ -29,14 +32,29 @@
 
     @Getter private String achievementName;
     @Getter @Setter private String channel;
-    @Getter @Setter private String processedMessage;
```

Comment:
the getter and setter for this should be deprecated, see https://github.com/DiscordSRV/DiscordSRV/blob/develop/src/main/java/github/scarsz/discordsrv/api/events/DeathMessagePostProcessEvent.java#L60-L68

Reply:
Should be added, let me know if it is done correctly